server/routes/temporary-accommodation/manage.ts

```diff
@@ -260,6 +261,9 @@ export default function routes(controllers: Controllers, services: Services, rou
       },
     ],
   })
+  get(paths.bookings.search.pattern, bookingSearchController.index(), {
+    auditEvent: 'VIEW_BOOKINGS_LIST',
```

Comment:
I'd be tempted to make this `VIEW_SEARCH_BOOKINGS` to keep the vocabulary consistent